web-client/src/presenter/utilities/singletonSequenceDecorator.ts

```diff
@@ -0,0 +1,16 @@
+import { setSingletonSequenceNameAction } from '@web-client/presenter/actions/setSingletonSequenceNameAction';
+import { unsetSingletonSequenceNameAction } from '@web-client/presenter/actions/unsetSingletonSequenceNameAction';
+
+export const singletonSequenceDecorator = (
+  sequenceName: string,
+  actionsList: any[],
+): any[] => {
+  return [
+    () => ({ sequenceName }),
+    setSingletonSequenceNameAction,
+    {
+      alreadyRunning: [],
```

Comment:
Is it worth logging a warning message here to flag devs that the same sequence is being run in cases they didn't expect?